kubernetes/package-kafka/kafkapkginstaller.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: kafkapkginstaller
+  namespace: openwhisk
+  labels:
+    name: kafkapkginstaller
+spec:
+  template:
+    metadata:
+      labels:
+        name: kafkapkginstaller
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: kafkapkginstaller
+        imagePullPolicy: IfNotPresent 
+        image: openwhisk/kafkapkginstaller
+        env:
+        - name: "APIHOST_URL"
+          value: "nginx.openwhisk"
+        - name: "DB_URL"
+          value: "http://whisk_admin:some_passw0rd@couchdb.openwhisk:5984"
+        - name: "DB_PREFIX"
+          value: "mq_prefix"
+        - name: "APIHOST"
+          value: "nginx.openwhisk"
+        - name: "AUTH"
+          value: "789c46b1-71f6-4ed5-8c54-816aa4f8c502:abczO3xZCLrMN6v2BKK1dXYFpXlPkccOFqm12CdAsMgRU4VrNZ9lyGVCGuMDGIwP"
```

Comment:
Mostly get these from configmaps.  
Use $(COUCHDB_SERVICE_HOST):$(COUCHDB_SERVICE_PORT_COUCHDB) to avoid needing dns.
Might need to refactor the init script a little bit because DB_URL concatenates information that is stored in a secret + some other envars.